holonix/README.md

```diff
@@ -45,3 +45,14 @@ inputs = {
 please see the following examples to learn more about common and more specific use cases:
 
 * [specifying custom component versions](examples/custom_versions/flake.nix)
+
+## Customizing the holochain binary build parameters
+
+The top-level flake output `packages.holochain` and `devShells.holonix` are customisable by means of [nixpkgs.lib.makeOverridable](https://nixos.org/manual/nixpkgs/stable/#sec-lib-makeOverridable).
+
+### Example: pass `--features chc` to holochain's `cargo build` command
+
+This means that you can pass e.g. `holochain.override { cargoExtraArgs = " --feature chc"; }` or any other desirable attribute to override the attributes that are passed to [craneLib.buildPackage](https://crane.dev/API.html#cranelibbuildpackage).
+
+Creating a devShell based on holonix this, can be achieved by passing `holonix.override { holochainOverrides = { cargoExtraArgs = "--features chc"; }}`.
```

Comment:
well i wish i could retain the funnyness and make it understandable too :laughing: for now i'll focus on the latter. how about this?

```suggestion
Creating a devShell based on holonix, passing on overrides to the holochain package can be achieved by passing for example `holonix.override { holochainOverrides = { cargoExtraArgs = "--features chc"; }}`.
```

Reply:
Ah, now I understand what you wanted to say. I think the latter was almost understandable then =) I suggest
```suggestion
In a devShell based on holonix, this can be achieved by defining `holonix.override { holochainOverrides = { cargoExtraArgs = "--features chc"; }}`.
```